meta/proposals.md

```diff
@@ -0,0 +1,98 @@
+# Proposals
+
+This document explains what proposals are and why it's important that we spend
+the time writing & sharing them. We also have a [template for proposals](template.md).
+
+## What are proposals?
+
+Developers produce code. Program managers (PM) produce proposals. It's an over
+simplification and doesn't cover what each discipline does, but it's a good way
+to think about the deliverables each discipline has:
+
+* **Developers** are expected to produce working code. This includes defining
+  and building an architecture, authoring unit tests and reviewing code changes.
+  Most people have a good working understanding of that role.
+
+* **Program managers** on the other hand is a role that is harder to define.
+  They aren't managers in the traditional sense (meaning they don't have people
+  reporting to them) and they are clearly not programmers (in the sense that
+  they are writing code that is shipped). The job of a program manager is
+  defining the product and driving the feature work to make the vision a
+  reality. Program manager usually don't come up with the vision by themselves
+  but instead define them as a result of requirements that they gathered from
+  customers. In some cases that's done via face-to-face interaction, in other
+  cases via market research. Program managers are engineers, but they don't
+  engineer the product's implementation but rather the product's public face.
+
+Most of you have heard the phrase "don't build what the customer asks for, build
+what the customer needs". The job of a program manager is to do the translation
+from "here is a problem a customer has" to "here is the set of features our
+product needs in order to make our customers successful". Proposals are the
+output of such a translation and capture the identified problems and how the
+feature (or set of features) will allow the customer to solve them.
+
+Of course, this doesn't mean that only PMs are allowed to write proposals;
+anyone inside or outside of Microsoft is welcome to submit them. But as it is
+with code: we have a bar for what constitutes a good proposal and whether we're
+willing to accept it.
+
+We do have a [template for proposals](template.md).
+
+## Why do we need proposals?
+
+There seems to be a general trend in our industry to move away from rigid rules
+and processes in order to be flexible and adaptive. For many people that seems
+to indicate that any form of written product definition that isn't source code
+is a waste of time and instead should be spent on coding.
+
+But reduction of process and paper work isn't a goal in and of itself. The goal
+is to generate customer value faster in order to adapt to an ever faster
+changing world. There is no point in doing process for the sake of process,
+that's overhead. No customer ever said "I lover your product because your
+engineering process is so pristine". But equally, we need to avoid wasting time
```

Comment:
While I believe this English rule is illogical, officially the period goes inside the quotes.